environment.yaml

```diff
@@ -0,0 +1,57 @@
+name: pyinknife2-env
+channels:
+  - conda-forge
+dependencies:
+  pip:
+    - python==3.7
+    - attrs==20.3.0
+    - biopython==1.78
+    - click==8.0.1
+    - cloudpickle==1.6.0
+    - cycler==0.10.0
+    - dask==2021.5.0
+    - decorator==4.4.2
+    - distributed==2021.5.0
+    - fsspec==2021.5.0
+    - funcsigs==1.0.2
+    - cython==0.29.23
+    - GridDataFormats==0.5.0
+    - gsd==2.4.2
+    - HeapDict==1.0.1
+    - importlib-metadata==4.0.1
+    - iniconfig==1.1.1
+    - joblib==1.0.1
+    - kiwisolver==1.3.1
+    - locket==0.2.1
+    - matplotlib==3.4.1
+    - MDAnalysis==1.1.1
+    - mmtf-python==1.1.2
+    - mock==4.0.3
+    - msgpack==1.0.2
+    - networkx==2.5.1
+    - numpy==1.20.2
+    - packaging==20.9
+    - pandas==1.2.4
+    - partd==1.2.0
+    - Pillow==8.2.0
+    - psutil==5.8.0
+    - pkg-resources==0.0.0
```

Comment:
this is here because of a bug in Debian-based distros and can be removed

Reply:
Ok perfect.